liby2/src/Y2ProgramComponent.cc

```diff
@@ -419,7 +421,7 @@ void Y2ProgramComponent::sendToExternal(const string& value)
     // result (..) from the input pipe, which is very important. Otherwise the
     // result value would be dropped.
 
-    write(to_external[1], "\n", 1);
+    (void) (write(to_external[1], "\n", 1) == 1);
```

Comment:
I thought ignoring the result was justified by the explanation above the code, but I admit that we can simply log an error. Will do.